website/server/libs/highlightMentions.js

```diff
@@ -112,7 +122,10 @@ function findTextBlocks (text) {
     const match = targetText.match(regex);
 
     if (!match) {
-      // Should not happen, but insert to handle bugs gracefully
+      logger.error(
+        new Error('Failed to match source-mapping regex to find ignore block'),
+        { text, targetText, regex },
```

Comment:
this looks good but can we convert the regex to a string (String(regex) should work), we've had some issues in the past with the logging not working well with uncommon data types and anyway it'll be converted to JSON. This way we'll be sure we're getting a string logged

Reply:
Ah yeah, it looked fine locally with the Console transport configured but I don't know how it behaves with the production transports of course. Will fix 👍

Reply:
Fixed in fbcb25d8288d1e83d09c53a99a0c7fee20c0f70a